src/main/java/core/basesyntax/service/FruitParser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.model.Fruit;
+
+public interface FruitParser {
```

Comment:
```suggestion
public interface FruitMapper {
```
mapper is a better name for this class

Reply:
done